account/urls.py

```diff
@@ -1,10 +1,21 @@
 from django.urls import path
 from .views      import UserCheckView, SignUpView, SignInView, SocialSignInView, CollectionView
+from .views      import (
+    UserCheckView,
+    SocialSignInView,
+    SignUpView,
+    SignInView,
+    CollectionView,
+    PlaylistView,
+    AddToPlaylistView
+)
 
 urlpatterns = [
     path('/', UserCheckView.as_view()),
     path('/signup', SignUpView.as_view()),
     path('/signin', SignInView.as_view()),
     path('/collection', CollectionView.as_view()),
+    path('/playlist', PlaylistView.as_view()),
+    path('/add_to_playlist', AddToPlaylistView.as_view()),
```

Comment:
엔드포인트명과 클래스명에 보통 동사를 붙이지는 않아서 하기와 같이 변경하는것 어떤가요 ? 
```suggestion
    path('/playlist-track', PlaylistTrackView.as_view()),
```